subsys/bluetooth/mesh/prov_device.c

```diff
@@ -640,7 +644,7 @@ int bt_mesh_prov_disable(bt_mesh_prov_bearer_t bearers)
 
 	if (IS_ENABLED(CONFIG_BT_MESH_PB_GATT) &&
 	    (bearers & BT_MESH_PROV_GATT)) {
-		bt_mesh_proxy_prov_disable(true);
+		(void)bt_mesh_proxy_prov_disable();
```

Comment:
Should also disconnect?

Reply:
Currentlly in pb_gatt disconnect cb --> bt_mesh_pb_gatt_close --> link_closed --> cb->link_closed --> reset_state() --> Clear LINK_ACTIVE.

And we add
`	if (bt_mesh_prov_active()) {
		return -EBUSY;
	}` before to avoid this